hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/view/HtmlBlock.java

```diff
@@ -97,4 +97,28 @@ protected UserGroupInformation getCallerUGI() {
     return callerUGI;
   }
 
+  /**
+   * Initialize User Help Information Div.
+   * When the user does not configure the Yarn Federation function, prompt the user.
+   *
+   * @param html html page
+   * @param isEnabled true, Federation is enabled; false, Federation is not enabled
```

Comment:
```
   * @param html HTML page.
   * @param isEnabled If federation is enabled.
```

Reply:
Thank you very much for your help reviewing the code, I will fix it.